src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -328,6 +373,227 @@ public void SetFocus(bool focused)
+        /// <since_tizen> 6 </since_tizen>
+        public void SendOrientation (Orientation orientation)
+        {
+            Interop.ChromiumEwk.ewk_view_orientation_send(_realHandle, (int)orientation);
+        }
+
+        /// <summary>
+        /// Suspends the operation associated with the view.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void Suspend ()
+        {
+            Interop.ChromiumEwk.ewk_view_suspend(_realHandle);
+        }
+
+        /// <summary>
+        /// Resumes the operation associated with the view.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void Resume ()
+        {
+            Interop.ChromiumEwk.ewk_view_resume(_realHandle);
+        }
+
+        /// <summary>
+        /// Gets the current scale factor of the page.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public double Scale
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_view_scale_get(_realHandle);
+            }
+        }
+
+        /// <summary>
+        /// Sets the current scale factor of the page.
+        /// </summary>
+        /// <param name="scaleFactor,">A new level to set.</param>
+        /// <param name="p">The class Point object with X, Y coordinates.</param>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool SetViewScale (double scaleFactor, Point p)
+        {
+            return Interop.ChromiumEwk.ewk_view_scale_set(_realHandle, scaleFactor, p.X, p.Y);
+        }
+
+        /// <summary>
+        /// Sets the current page's visibility.
+        /// </summary>
+        /// <param name="enable">'true' to set on the visibility of the page, 'false' otherwise.</param>
+        /// <returns>'true' on success, otherwise 'false'.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool SetViewVisibility (bool enable)
+        {
+            return Interop.ChromiumEwk.ewk_view_visibility_set(_realHandle, enable);
+        }
+
+        /// <summary>
+        /// Get and Sets the scroll position of the page.
+        /// </summary>
+        /// <returns>The class Point object with X, Y coordinates.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public Point ScrollPos
+        {
+            get
+            {
+                Point p;
+                Interop.ChromiumEwk.ewk_view_scroll_pos_get(_realHandle, out p.X, out p.Y);
+                return p;
+            }
+            set
+            {
+                Interop.ChromiumEwk.ewk_view_scroll_set(_realHandle, value.X, value.Y);
+            }
+        }
+
+        /// <summary>
+        /// Scrolls the webpage of view by x and y.
+        /// </summary>
+        /// <param name="p">The class Point object with X, Y coordinates.</param>
+        /// <since_tizen> 6 </since_tizen>
+        public void ScrollBy (Point p)
```

Comment:
Plz change the name of param to more specific one.

Reply:
Done